packages/preview/muw-touying/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "muw-touying"
+version = "0.1.0"
+authors = ["Emberwhirl <n12440050@students.meduniwien.ac.at>"]
+repository = "https://github.com/Emberwhirl/muw-touying"
+description = "Touying theme for Medical University of Vienna"
+license = "MIT"
+entrypoint = "lib.typ"
+keywords = ["touying", "presentation", "slides", "muw", "Medizinische Universität Wien", "Medical University of Vienna", "meduni", "wien", "vienna", "academic", "clinical"]
+categories = ["presentation", "office"]
+exclude = ["examples/"]
```

Comment:
The error that the CI couldn't report is about this line: the template directory should not be excluded (unfortunately, GitHub can't display errors directories, only on individual files).

Reply:
Thanks for the clarification. I guess there is no need to exclude any. However CI is still reporting errors on the files under the `examples` directory.

Reply:
Thanks for the clarification. I guess there is no need to exclude any files so I deleted this line in `typst.toml`. However CI is still reporting errors on the files under the `examples` directory, giving message that the files in `examples` should not be excluded